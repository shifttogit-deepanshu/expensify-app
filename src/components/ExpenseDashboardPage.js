import React from 'react'
import ExpenseList from "./ExpenseList"
import ExpenseListFilters from "./ExpenseListFilters"

const ExpenseDashboardPage = ()=>{
    return <div>
    <h2>This is ExpenseDashboardPage</h2>
    <ExpenseListFilters />
    <ExpenseList />
    </div> 
}

export default ExpenseDashboardPage;