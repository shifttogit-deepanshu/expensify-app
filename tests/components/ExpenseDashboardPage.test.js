import React from 'react'
import {shallow} from "enzyme"
import ExpenseDashboardPage from "../../src/components/ExpenseDashboardPage"

test('should render header correctly',()=>{
    const wrapper = shallow(<ExpenseDashboardPage/>)
    expect(wrapper).toMatchSnapshot()

})

