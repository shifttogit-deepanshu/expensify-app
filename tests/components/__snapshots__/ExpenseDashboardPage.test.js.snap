// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<div>
  <h2>
    This is ExpenseDashboardPage
  </h2>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
